devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -102,16 +107,13 @@ def updated() {
 	log.debug "updated"
 // Device wakes up every 4 hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
-	response(getConfigurationCommands())
+	getConfigurationCommands()
 }
 
 def configure() {
-	if (isEverspringSP815orSP817()) {
+	if (isEverspringSP817()) {
 		state.configured = false
 		state.intervalConfigured = false
```

Comment:
nit: you only need one configuration flag now, right?

Reply:
right! fixed